packages/paste-website/src/pages/components/button/index.mdx

```diff
@@ -345,6 +299,65 @@ styles. No other Button functionality is reset.
 </Button>`}
 </LivePreview>
 
+#### Icon Buttons
+
+Use primary icon, secondary icon, and destructive icon Button variants for buttons
+that, for a specific reason and purpose, can only have an icon in them, such as
+compact UI situations. These variants do not have any border nor background color.
+When using these variants, use `size="reset"` in the Button component to remove
+any unnecessary padding. To change the size of an icon button, change the size of the Icon component itself.
```

Comment:
```suggestion
any unnecessary padding. To change the size of an icon Button, change the size of the icon component itself.
```

Reply:
take these suggestions or leave them... not exactly sure whether we should capitalize these or not but I do think we should be consistent